.github/workflows/update-flake.yml

```diff
@@ -27,13 +27,12 @@ jobs:
         run: ./update-flake.sh
 
       - name: Get access token
-        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a # v2.1.0
+        uses: actions/create-github-app-token@df432ceedc7162793a195dd1713ff69aefc7379e # v2.0.6
         id: generate-token
         with:
-          app_id: ${{ secrets.LICENSING_APP_ID }}
-          installation_retrieval_mode: "id"
-          installation_retrieval_payload: ${{ secrets.LICENSING_APP_INSTALLATION_ID }}
-          private_key: ${{ secrets.LICENSING_APP_PRIVATE_KEY }}
+          # Get token for app: https://github.com/apps/tailscale-license-updater
+          app-id: ${{ secrets.LICENSING_APP_ID }}
+          private-key: ${{ secrets.LICENSING_APP_PRIVATE_KEY }}
```

Comment:
This token is used to create a PR in this self-same repository, so we don’t need the specify the installation ID.